### PR TITLE
NYT article page project

### PR DESCRIPTION
**Pull request by nusretbutunay**
This assignment consists of making a clone of a New York Times article page.

### CODE REVIEW

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

File: style.css [hunk 1]
```diff
@@ -0,0 +1,578 @@
...
+#expand-btn i {
+  margin: 0 30px;
+  font-size: 1.2em;
+}
+
+#inflation-grid-container img {
+  height: 150px;
+}
+
+#inflation-grid-container span {
```
Comment (andresporras3423) [inline style.css]: Don't overuse ID Selectors

File: style.css [hunk 2]
```diff
@@ -0,0 +1,578 @@
...
+  color: var(--secondary-text-color);
+}
+
+.more-articles-grid h3 a:hover {
+  text-decoration: underline;
+}
+
+footer .footer-list li a:hover {
+  text-decoration: underline;
+}
```
Comment (andresporras3423) [inline style.css]: Don't repeat yourself, in order to minimize code you can group selectors with the same style definition (in your CSS files).

File: style.css [hunk 3]
```diff
@@ -0,0 +1,578 @@
...
+  font-style: italic;
+}
+
+#article-header figure {
+  width: 100%;
+}
+
+#article-header figure img {
+  width: 100%;
+}
```
Comment (andresporras3423) [inline style.css]: Don't repeat yourself, in order to minimize code you can group selectors with the same style definition (in your CSS files).

File: style.css [hunk 4]
```diff
@@ -0,0 +1,578 @@
...
+
+#article-header figure figcaption span {
+  font-size: 12px;
+
+  /* color: #888; */
+  color: var(--secondary-text-color);
+}
+
+#video {
```
Comment (andresporras3423) [inline style.css]: Don't overuse ID Selectors

File: style.css [hunk 5]
```diff
@@ -0,0 +1,578 @@
...
+  margin-top: 30px;
+}
+
+#author h3 {
+  padding: 2% 0;
+}
+
+#author h3 a:hover {
+  text-decoration: none;
+}
```
Comment (andresporras3423) [inline style.css:357]: You have a similar style in another selector on line 219

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @nusretbutunay @mohapakram** 
Good job with your first submission for this project :muscle:  
your page looks really similar to the original :clap: :clap: :clap:  
Your project passes most of the requirements. There are just a few details to improve before you can go ahead:
- [ ] Don't overuse ID Selectors
- [ ] Don't repeat yourself, in order to minimize code you can group selectors with the same style definition (in your CSS files).
- [x]  Check over the "editors pick" in the original page and in your page. In the original:




In your page:



Add the "Subscribe to The Times" section.

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras
(for any question you can contact me in slack as @oscarporras02)

**Review by trekab — APPROVED** [review]
# Approved! :tada: 
Hi @nusretbutunay and @mohapakram! Well done! :+1: You may merge to master, submit the project completion form, and proceed to your next project.

All the best on your next project! :wink: 
__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Pull request merged**